images/ose-frr.yml

```
@@ -15,6 +15,7 @@ enabled_repos:
 for_payload: false
```

also since it's an operand it should have the operator as a dependent:
```suggestion
for_payload: false
dependents:
- ose-metallb-operator
```